collect_app/src/androidTest/java/org/odk/collect/android/support/FormActivityTestRule.kt

```diff
@@ -0,0 +1,77 @@
+package org.odk.collect.android.support
+
+import android.app.Activity
+import android.app.Application
+import android.content.Intent
+import androidx.test.core.app.ActivityScenario
+import androidx.test.core.app.ApplicationProvider
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+import org.odk.collect.android.activities.FormEntryActivity
+import org.odk.collect.android.external.FormsContract
+import org.odk.collect.android.injection.DaggerUtils
+import org.odk.collect.android.storage.StorageSubdirectory
+import org.odk.collect.android.support.pages.FormEntryPage
+import org.odk.collect.projects.Project
+import org.odk.collect.projects.Project.Companion.DEMO_PROJECT
+import java.io.IOException
+
+class FormActivityTestRule : TestRule {
+    private val formFilename: String
+    private val formName: String
+    private val mediaFilePaths: List<String>?
+    private var formEntryPage: FormEntryPage? = null
+
+    constructor(formFilename: String, formName: String) {
```

Comment:
Now as it's kotlin can we use one primary constructor instead of those two?

Reply:
Yes, done.